dev-requirements.txt

```diff
@@ -2,3 +2,4 @@ https://github.com/cloudify-cosmo/cloudify-dsl-parser/archive/master.zip
 https://github.com/cloudify-cosmo/cloudify-rest-client/archive/master.zip
 https://github.com/cloudify-cosmo/cloudify-plugins-common/archive/master.zip
 pyzmq==15.1.0
+psutil>=2.0.0
```

Comment:
OK, if I can't do it this way then I'll pin to latest version (5.2.1) as that's what I've been testing against.